### DISCUSSION TITLE
Is android version even the same app?

### DISCUSSION THREAD

**Discussion by primski** [transmission/transmission#7016] [General]
Where are speed limit settings?
Why does it stop download and won't restart unless i kill the app and start again?
Why doesn't it find more peers?

Using it on android tv.
v2.9.4

**Comment by Pentaphon** [comment]
Can you link to the exact software you are using?

**Comment by primski** [comment]
[This one](https://play.google.com/store/apps/details?id=com.ap.transmission.btc&hl=en). Am using it on Android TV.

**Reply by Pentaphon** [reply]
I would recommend an app that was actually designed for Android TV like [this one.](https://play.google.com/store/apps/details?id=com.biglybt.android.client)

Plus that app is very much out of date so it you have to torrent on Android TV you might as well use an updated torrent client.

**Comment by primski** [comment]
Transmission is perfectly fine torrent client. I don't need all bells and whistless just to download a few things, certainly not in-app purchases. Thanks though.

I have noticed it hasn't been updated in a while but only android version. Perhaps a task for the developer.

**Reply by Pentaphon** [reply]
That app of yours has probably been abandoned. BiglyBT works well on Android TV plus it is open source. Unless Transmission gets a better port or an official Android port, you should probably just use BiglyBT.

**Reply by peerless2012** [reply]
Does this repo support android already?
